DoorGate.xcodeproj/project.pbxproj

```diff
@@ -16,6 +16,11 @@
 		926D4D212012546200F3CB4C /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 926D4D1F2012546200F3CB4C /* LaunchScreen.storyboard */; };
 		926D4D2C2012546200F3CB4C /* DoorGateTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 926D4D2B2012546200F3CB4C /* DoorGateTests.swift */; };
 		926D4D372012546200F3CB4C /* DoorGateUITests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 926D4D362012546200F3CB4C /* DoorGateUITests.swift */; };
+		92FC9E91201B90BB00FAE803 /* DoorState.swift in Sources */ = {isa = PBXBuildFile; fileRef = 92FC9E90201B90BB00FAE803 /* DoorState.swift */; };
```

Comment:
is this still generated code? Does this file typically get checked in?

Reply:
good catch, i thought i had deleted this file in xcode.